sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducer/EventHubProducerTests.cs

```diff
@@ -316,7 +316,7 @@ public async Task SendWithoutOptionsInvokesTheTransportProducer()
 
             (var calledWithEvents, var calledWithOptions) = transportProducer.SendCalledWith;
 
-            Assert.That(calledWithEvents, Is.SameAs(events), "The events should be the same instance.");
+            CollectionAssert.AreEqual(calledWithEvents, events, "The events should contain same elements.");
```

Comment:
Can we please stick with the `Assert.That` for mat.   I believe that the `Is.EquivilantTo` constraint is what you're looking for here.

Reply:
Thank you!

Reply:
That's what it was!